package.json

```diff
@@ -66,6 +73,12 @@
     "react-dom": "^16.4.0",
     "react-highlight": "^0.12.0",
     "rimraf": "^2.6.2",
+    "rollup": "^0.64.1",
```

Comment:
Adding Rollup.js and plugins.  While some of the plugins mirror what we were doing with Webpack, it seems to be the standard for React/UI/Module libraries (see Material UI, Bootstrap, etc).